content/docs/command-reference/exp/branch.md

```diff
@@ -15,17 +15,19 @@ positional arguments:
 
 ## Description
 
-Makes a
-[Git branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
-off the last commit (`HEAD`) based on the given `experiment`, using the `branch`
-name provided. This makes the given experiment
-[persistent](/doc/user-guide/experiment-management#persistent-experiments) in
-the repo.
-
-In most cases this is similar to using `dvc exp apply` (applies the `experiment`
-results to the workspace) followed by Git branching and committing, except that
-`dvc exp branch` **does not** switch into the created `branch`. Equivalent to
-this:
+Makes a named Git
+[`branch`](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
+containing the target `experiment` (making it
+[persistent](/doc/user-guide/experiment-management#persistent-experiments)). The
+branch is based on the commit that was active (`HEAD`) at the time that the
+target experiment was run.
```

Comment:
I don't think there's anything wrong with this sentence, but I found it a little confusing. Is this necessary? Seems more like a general explanation of experiments than anything specific to `exp branch`, but I might be missing the point you're trying to convey.

Reply:
Yeah maybe it doesn't belong in the first paragraph. But I think it's key that people understand that the branch won't be based on their current `HEAD`. BTW the sample `git` commands below are still incorrect, I see... ⌛

Reply:
Changed a bit 2685e741 for now, but I may change it per your other feedback...